index.js

```diff
@@ -37,18 +87,42 @@
  *
  * And returns the first and last name concatenated together (i.e. 'Bob Evans').
  */
+ function getFullName(firstName, lastName) {
+ 	var fullName = {
+ 		firstName: 'Bob',
+	 	lastName: 'Evans'
+	 };
+	 console.log(fullName.firstName + ' ' + fullName.lastName);
+	 return fullName.firstName + ' ' + fullName.lastName;
+ }
+ getFullName();
```

Comment:
Here, we do not want to define the object within the function, we want to accept it as input to the function through a parameter. Let's call that parameter `person` since it represents a person with a name:

```js
function getFullName(person) {
    return person.firstName + ' ' + person.lastName;
}
```

Now, we can call this function using the Bob Evans object as well as any other object.

```js
var bobEvans = {
    firstName: 'Bob',
    lastName: 'Evans'
};
getFullName(bobEvans);

var person2  = {
    firstName: 'Kent',
    lastName: 'Miller'
};
getFullName(person2);

getFullName({firstName: 'Jacob', lastName: 'Friedmann'});
```